### PR TITLE
Add blog post for KubeDB release v2023.08.18

### PR DESCRIPTION
**Pull request by raihankhan**
Signed-off-by: raihankhan <raihan@appscode.com>

### CODE REVIEW

File: content/post/kubedb-v2023.08.18/index.md [hunk 1]
```diff
@@ -0,0 +1,234 @@
...
+- security
+---
+
+We are pleased to announce the release of KubeDB v2023.08.18. This post lists all the major changes done in this release since the last release. The release includes -
+
+- **Use of the restricted pod security label** ⇒  Pod security policies has been removed in k8s 1.25. In place, It brings `Pod Security Standards` into the picture. We are using the restricted mode (https://kubernetes.io/docs/concepts/security/pod-security-standards/#restricted) to all of the namespaces where our operators will be installed.   Thus, we achieve some good security standards, like running as non-root-user, privilege escalation will not happen, some destructive kernel capabilities will be dropped etc.
+
+- **Uniform conditions across database opsRequests** ⇒ We utilize the status.Conditions section of the opsRequest CR for correctly maintaining the steps of an opsRequest like VersionUpdate, HorizontalScaling etc. These conditions have been made uniform in all of our supported databases now.
+
+- **Reduce get/patch api calls** ⇒ #TODO
```
Comment (ArnobKumarSaha) [inline content/post/kubedb-v2023.08.18/index.md:37]: todo ?

File: content/post/kubedb-v2023.08.18/index.md [hunk 2]
```diff
@@ -0,0 +1,234 @@
...
+        storage: 1Gi
+```
+
+**Fixes:** Use --bind_ip to fix version `3.4.*` CrashLoopbackOff issue.
+Fix for MongoDB HorizontalScale down for shared cluster also have been addressed in this release.
+
+## MySQL:
+
+
+#TODO
```
Comment (ArnobKumarSaha) [inline content/post/kubedb-v2023.08.18/index.md:170]: ?

**Review by ArnobKumarSaha — CHANGES_REQUESTED** [review]

File: content/post/kubedb-v2023.08.18/index.md [hunk 3]
```diff
@@ -0,0 +1,234 @@
...
+- percona-xtradb
+- pgbouncer
+- postgresql
+- prometheus
+- proxysql
+- redis
+- security
+---
+
+We are pleased to announce the release of KubeDB v2023.08.18. This post lists all the major changes done in this release since the last release. The release includes -
```
Comment (DRoyyy) [inline content/post/kubedb-v2023.08.18/index.md:31]: Use redirect link:
We are pleased to announce the release of [KubeDB v2023.08.18](https://kubedb.com/docs/v2023.08.18/setup/).

File: content/post/kubedb-v2023.08.18/index.md (hunk 1 above)
Comment (raihankhan) [reply content/post/kubedb-v2023.08.18/index.md:37]: The description for Reducing Get/Patch calls changes is not added yet.

File: content/post/kubedb-v2023.08.18/index.md (hunk 2 above)
Comment (raihankhan) [reply content/post/kubedb-v2023.08.18/index.md:170]: The description for MySQL changes is not added yet.

**Review by heheh13 — DISMISSED** [review]

**Review by ArnobKumarSaha — APPROVED** [review]

**Pull request merged**